pkg/resources/server.go

```diff
@@ -315,18 +303,18 @@ func (m *RdmaDevPlugin) cleanup() error {
 }
 
 // GetInfo get info of plugin
-func (m *RdmaDevPlugin) GetInfo(ctx context.Context, rqt *registerapi.InfoRequest) (*registerapi.PluginInfo, error) {
+func (m *resourceServer) GetInfo(ctx context.Context, rqt *registerapi.InfoRequest) (*registerapi.PluginInfo, error) {
 	pluginInfoResponse := &registerapi.PluginInfo{
 		Type:              registerapi.DevicePlugin,
 		Name:              m.resourceName,
-		Endpoint:          filepath.Join(sockDir, m.socketName),
+		Endpoint:          filepath.Join(activeSockDir, m.socketName),
```

Comment:
why was this changed to activeSockDir ?
when creating a resource server i see: sockDir is intialized.

Reply:
Changed the name to be more clear

Reply:
use m.socketPath